pkg/dogstatsd/enrich.go

```diff
@@ -8,21 +8,24 @@ import (
 )
 
 var (
-	hostTagPrefix       = "host:"
-	entityIDTagPrefix   = "dd.internal.entity_id:"
-	entityIDIgnoreValue = "none"
+	hostTagPrefix        = "host:"
+	entityIDTagPrefix    = "dd.internal.entity_id:"
+	entityIDIgnoreValue  = "none"
+	cardinalityTagPrefix = "dd.internal.cardinality:"
```

Comment:
This makes sense yes. Especially if we plan to contribute to the official dsd clients, all of this would become hidden and users won't have to deal directly with the tag name and the keys/values.
I'll update the PR accordingly!